.circleci/config.yml

```diff
@@ -4,9 +4,9 @@ version: 2.1
 #        EXECUTORS
 # -------------------------
 executors:
-  node8:
+  node10:
```

Comment:
CI Node image has been updated to the LTS in #2140.

Reply:
Cool. Maybe it would solve the memory issue 😝

Reply:
@slorber LTS bump fixed that OOM issue it seems 🙂

Reply:
Thank you for your help! 😄